src/cloud/env/environments-start.md

```diff
@@ -15,28 +15,7 @@ To manage the branches and environments with the Project Web Interface, see [Man
 
 ## Common Magento Cloud CLI commands {#env-start-comm}
```

Comment:
```suggestion
## Use Magento Cloud CLI commands {#env-start-comm}
```

Reply:
Not so fond of "Use", but since we redirected to the CLI reference, the current heading needs an update.